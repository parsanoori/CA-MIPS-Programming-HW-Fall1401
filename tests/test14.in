2
181145481
712997553