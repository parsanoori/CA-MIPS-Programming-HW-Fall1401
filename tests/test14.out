181145481
712997553